app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AdminController < ApplicationController
+
+  def index
+    unless current_user_is_admin?
+      flash[:error] = "You do not have sufficient permissions to view this page"
```

Comment:
Is there an exception for spitting out an access denied kind of error in Hydra?  I thought there was, but I can't recall.  It would be nice to ensure we always handle access denied errors the same way, though.